zebrad/src/commands/tests.rs

```diff
@@ -0,0 +1,41 @@
+use clap::Parser;
+
+use crate::commands::ZebradCmd;
+
+use super::EntryPoint;
+
+#[test]
+fn args_with_subcommand_pass_through() {
+    let test_cases = [
+        (true, false, vec!["zebrad"]),
+        (true, true, vec!["zebrad", "-v"]),
+        (true, true, vec!["zebrad", "--verbose"]),
+        (true, false, vec!["zebrad", "-h"]),
+        (true, false, vec!["zebrad", "--help"]),
```

Comment:
Should these help cases use the start command? I'm pretty sure they used to just show the help.

Reply:
> Should these help cases use the start command?

No, and they don't, this test is wrong.